### PR TITLE
Hut fire Italian translation

### PR DESCRIPTION
**Pull request by AndySky21**
This adds Italian translations for hut fire related strings, as well as translation corrections (see Italian translation fix). Sorry, the 2 things could not be separated: however, as JS file consists of only one line, I suppose it would flag it as "unable to automatically merge" if you tried to correct Italian translation and then to apply new strings to the old list. So, either you only take the translation fix and discard this, or simply merge this for both (trust me, some of those fixes *are* necessary).

### CODE REVIEW

File: lang/it/strings.po [hunk 1]
```diff
@@ -77,265 +83,269 @@ msgstr "salvato correttamente nella memoria dati in Dropbox"
...
+#: script/engine.js:17
 msgid "punches do more damage"
-msgstr "i pugni causano danno maggiore"
+msgstr "i pugni causano più danni"
 
-#: script/engine.js:24
+#: script/engine.js:20
 msgid "learned to throw punches with purpose"
-msgstr "hai appreso come colpire in modo efficace"
+msgstr "hai appreso come tirare pugni in modo efficace"
```
Comment (nicholaspellizer) [inline lang/it/strings.po]: secondo le specifiche narrative non bisognerebbe mai fare riferimento al giocatore "hai appreso", propongo: "i pugni ora colpiscono con maggiore determinazione"

Comment (AndySky21) [inline lang/it/strings.po]: "con maggiore efficacia", sarebbe forse meglio, perché la determinazione è caratteristica di chi li tira.
Quello che mi dici però è importante, perché quella frase non è la sola a rivolgersi al giocatore. Vanno corrette tutte.

**Comment by AndySky21** [comment]
Note to developers: is that true that notifications such as "learned etc." are not meant to refer to the character/player?
I often translated them with "hai appreso", which means "you've learned", leaving the impersonal form in the perk description (literally translating sentences such as "punches do more damage").
I also saw that in French the sentence "vous avez appris" (you've learned) is often used.

**Comment by Continuities** [comment]
It is meant to refer to the player character, but ADR avoids using any
personal pronouns as a matter of style. "You've learned some stuff" becomes
"learned some stuff." It's not exactly grammatical, but it's readable and
adds to the overall feel of the game. Not sure how this stylistic choice
translates in other languages. I wonder if it would be acceptable to just
write "aves appris" in French...

**Comment by AndySky21** [comment]
"avez appris" maybe isn't but, maybe "on a appris" could work (impersonal). That could be an issue for French translators.
Same for Italian, but impersonal form seems a bit forced.

**Comment by AndySky21** [comment]
A @nicholaspellizer: Che ne dici di 
`"si è appreso a colpire etc."`
per le notifiche? 
`"Appreso come colpire etc."` mi sembra sgradevole, a meno di non scrivere
`"Appreso: come colpire etc."` (con i due punti).
Il riferimento (non al giocatore, ma al personaggio) permette di distinguere la notifica nella barra sinistra dalla nota nel riquadro di dialogo.

**Comment by AndySky21** [comment]
Italian translation has been reviewed. All "learned" and similar impersonal sentences have been translated into impersonal Italian form ( @nicholaspellizer : impersonal i.e. "si è appreso come..."/"si è imparato a..." )
On my way towards impersonalisation, I changed a couple sentences to make them clearer.
Note: A Mysterious Wanderer would always render with a gender in Italian and so the male "generic" form always looks like the male wood-craving wanderer. In order to refer to a generic event, I translated it into plural for both the dialog title and the notification, so it is not too specific.
Also In This Commit: hut fire event texts (still missing).

**Comment by nicholaspellizer** [comment]
As the Wiki states: "Sentences never start with capital letters, and are as clipped as possible. Instead of using commas, try breaking your thought into two shorter sentences. Use "the" as little as possible.".

@AndySky21 : i due punti non mi sembrano adatti, sembra un log più che una notifica al giocatore. Concordo però che "Appreso come colpire" benché più vicino alla forma impersonale ricercata in inglese non suona bene per niente. L'impersonale preceduto dalla particella 'si' mi sembra un buon compromesso. Ho dato un occhiata veloce ai cambiamenti e mi sembra molto più in linea con l'atmosfera originale.

**Comment by AndySky21** [comment]
@nicholaspellizer speravo andasse bene. Diciamo che mi sono preso la responsabilità quasi da solo e sono andato avanti a testa bassa per la traduzione. Spero che ora vada bene!

I just hope that the presence of hut fire strings translation is correct, so this commit can be reviewed and merged!

File: lang/it/strings.po [hunk 2]
```diff
@@ -3205,60 +3226,79 @@ msgstr ""
...
+msgid "all residents in the hut perished in the fire."
+msgstr "tutti quelli che vivevano nella capanna sono morti nell'incendio"
+
+#: script/events/outside.js:77
+msgid "a fire has started"
+msgstr "è scoppiato un incendio"
+
+#: script/events/outside.js:87
+msgid "mourn"
+msgstr "piangi i defunti"
```
Comment (nicholaspellizer) [inline lang/it/strings.po]: "vengono compianti i morti" o un altra forma impersonale

Comment (AndySky21) [inline lang/it/strings.po]: No, quello è un pulsante, come "raccogli legname" o "controlla trappole". E' il comando che serve per chiudere la finestra di dialogo.

Comment (AndySky21) [inline lang/it/strings.po]: Someone doesn't want to see the hut taking fire in Italian (lol)

**Comment by skj3gg** [comment]
Is this ready to merge?

**Comment by AndySky21** [comment]
Yes, it is. 
@nicholaspellizer focused my attention on the fact that "Learned to..." needed to be translated impersonally, and now all those sentences are translated as such (he reviewed the form and said it works for him too).
Then he suggested me to tranlate impersonally "Mourn", the button which closes the hut fire dialog, and I pointed out that it's a command, just like "say goodbye", "buy", "learn".
The translation is ready now.

**Comment by nicholaspellizer** [comment]
Since I'm being mentioned here, I can confirm this is ready to be merged.

**Comment by Continuities** [comment]
Good enough for me!

**Pull request merged**